core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -82,6 +82,7 @@ object TransactionMarkerChannelManager {
       metadataCache,
       txnMarkerPurgatory,
       sendThread,
+      networkClient,
```

Comment:
Simiarly, since one `channelManager` only have one `channel`, could we just add the closure of the networkclient inside `channel.clear` (nit: maybe rename it to `close` as well).